app/components/graph-editor/drawable-element.ts

```diff
@@ -0,0 +1,275 @@
+// File: drawable-element.ts
+// Created by: CJ Dimaano
+// Date created: February 4, 2017
+
+
+import { FONT_SIZE } from "./defaults";
+import { GraphEditorCanvas, point, rect, size } from "./graph-editor-canvas";
+import { Drawable } from "./drawable";
+import { DrawableGraph } from "./drawable-graph";
+import {
```

Comment:
I think we should focus on readability. Short imports shouldn't need two lines, long imports can be too long if we don't break them up.